src/app/shared/filter-bar/label-filter-bar/label-filter-bar.component.ts

```diff
@@ -95,6 +95,13 @@ export class LabelFilterBarComponent implements OnInit, OnDestroy {
     return !target.toLowerCase().includes(filter.toLowerCase());
   }
 
+  hasNoLabels(filter: string): boolean {
+    if (this.allLabels === undefined || this.allLabels.length === 0) {
+      return true;
+    }
+    return this.allLabels.every((label) => this.filter(filter, label.name));
```

Comment:
Prefer to use .some and invert the boolean

As long as one label is shown, do not show the No Labels text

Reply:
I've made the change!